### PR TITLE
Add Stale FF docs

### PR DESCRIPTION
**Pull request by bttf**

### CODE REVIEW

File: docs/sidebars.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ const sidebars = {
           id: "features/scheduling",
           label: "Scheduling",
         },
+        {
+          type: "doc",
+          id: "features/stale-detection",
+          label: "Stale Detection",
```
Comment (august-growthbook) [inline docs/sidebars.js:68]: With "Stale Detection" I don't know what is going stale or what is being detected. I know space in the left sidebar is an issue, but I do see other 2-line subheadings so I think it's OK to lengthen this to "Stale Feature Flag Detection", which is much more descriptive and better for searching.

Comment (bttf) [reply docs/sidebars.js:68]: Just want to point out that this nav item in the left sidebar is nested under 'Features', so there is some context there. The title of the actual page is 'stale feature flag detection' so searches should still hit it. Does that clarification make it ok or would you still prefer it more explicit?

Comment (bttf) [reply docs/sidebars.js:68]: image

Comment (august-growthbook) [reply docs/sidebars.js:68]: How about "Stale Feature Detection"? I think we need "Feature" in the subheading but we can drop "flag". I'm seeing "Feature" in some of the other subheadings so "Stale Feature Detection" would fit right in.

Comment (bttf) [inline docs/sidebars.js:68]:
```suggestion
          label: "Stale Feature Detection",
```

**Review by august-growthbook — APPROVED** [review]

**Review by august-growthbook — APPROVED** [review]

**Pull request merged**